amanuensis/resources/institution/__init__.py

```diff
@@ -3,4 +3,40 @@
 	get_all_institutions,
     update_institution,
     add_institution,
-)
\ No newline at end of file
+)
+from amanuensis.config import config
+import urllib.request, json
+def api_request():
+    """
+    Makes a call to the Consolidated Screening List api of developer.trade.gov. Information returned in the dictionary
+    can be accessed by info_dict["results"], which is a list of dictionaries containing information about the legality
+    of interacting with the returned companies.
+    """
+    name = config["INSTITUTION_NAME"]
+    if(name == ""):
+        print("No institution name entered in config file! Returning empty dictionary")
+        return {}
+    try:
+        url = f"https://data.trade.gov/consolidated_screening_list/v1/search?name={name}"
+
+        hdr ={
+        # Request headers
+        'Cache-Control': 'no-cache',
+        'subscription-key': 'a7c2cd5a313d430c9df8bc7918f2e14b',
+        }
+
+        req = urllib.request.Request(url, headers=hdr)
```

Comment:
For consistency please use the same requests module we are using elsewhere `import requests`